elastic/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+***Fixed***:
+
+* [AGENT-10055] Correct checking instance type of get data response ([#15554](https://github.com/DataDog/integrations-core/pull/15554))
```

Comment:
```suggestion
* [AGENT-10055] Correct checking instance type of get data response. See [#15554](https://github.com/DataDog/integrations-core/pull/15554).
```

Reply:
Thanks @brett0000FF .

Pasting a comment from [another pr](https://github.com/DataDog/integrations-core/pull/15517#discussion_r1293082367) regarding new format for changelog entries.

Reply:
Great, thanks for the heads up 👍